test/connector/tcp/mssql/test

```diff
@@ -1,55 +1,6 @@
 #!/bin/bash -e
 
-while getopts :s: opt; do
-    case $opt in
-        s) SECRETLESS_HOST=${OPTARG};;
-       \?) echo "Unknown option -$OPTARG"; exit 1;;
-    esac
-done
-# If the secretless host is not explicitly set on the command line,
-# then use the default names (either secretless or secretless-dev)
-# for the secretless host. Automatically detect if we're devmode
-# based on the existence of the secretless-dev container.  We assume
-# that you started your workflow using `./dev` if you are developing,
-# and this command will use the secretless-dev container.
-if [[ -z $SECRETLESS_HOST ]]; then
-  export SECRETLESS_HOST=secretless
-  if [[ ! -z $(docker-compose ps -q secretless-dev) ]]; then
-    SECRETLESS_HOST=secretless-dev
-  fi
-fi
+echo -e "\nRunning tests\n"
 
-echo "Waiting for '$SECRETLESS_HOST' service to start"
-
-# single quotes are intentional:
-# shellcheck disable=SC2016
-docker-compose run --rm --no-deps test bash -ec '
-counter=0
-while ! wget --quiet --output-document - http://'$SECRETLESS_HOST':5335/ready > /dev/null 2>&1; do
-    if expr $counter = 5 > /dev/null; then
-      echo ""
-      echo "Timed out waiting for Secretless"
-      exit 1
-    fi
-    let "counter=$counter+1"
-    >&2 printf ". "
-    sleep 1
-done
-printf "\n"
-'
-
-echo "'$SECRETLESS_HOST' service is up - continuing "
-echo ""
-echo "Running tests"
-echo ""
-
-docker-compose run \
-  -e SECRETLESS_HOST="$SECRETLESS_HOST" \
-  --rm \
-  --no-deps \
+docker-compose exec -T \
   test bash -c "go test -v ./test/connector/tcp/mssql"
-
-# print secretless logs after tests run
-ret=$?
-docker-compose logs $SECRETLESS_HOST
```

Comment:
Do we no longer need to print logs from the secretless container? Might be handy to have for CI failures.

Reply:
I am actually not sure if I like this, but I will share what I was thinking and you can tell me if you think I should put it back anyway :)

Pros of adding them to script output:
- The Jenkins logs will show the container logs, and it might help when trying to understand a test failure

Cons of adding them to the script:
- Every time I run the scripts locally I get a flood of container logs in my terminal, which makes it hard to see the rest of the important info that I can't get from running `docker logs`
- When you run in Jenkins, there is a ton of info from the container logs that pollutes the log output, and even in the blue ocean view it's hard to distinguish the container log output from the rest of the test script output. It may make it harder to scan for pass/fail at a glance
- In practice I haven't seen an integration test failure that I didn't need to reproduce locally before fixing anyway, at which point I would have access to the container logs on my local machine

I do think having the logs in the Jenkins output may help with some edge case test failures, but the experience of using the scripts in the meantime is degraded enough for me that I prefer to leave this out. That being said, I'm not the one running them most of the time - so if you feel strongly this should stay, I can find a way to get something back in.

Reply:
Your bullet item in the `Pros` has me completely sold on leaving this out... and I very much agree with the "hard to distinguish the container log output from the rest of the test script output". It took me a while to get used to that stream of secretless log showing up after test results.